docker/run_docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -ex # Enable 'set -e' (exit on error) and 'set -x' (debugging) options
+
+TAG=moabb
+MOUNT_POINT="/home/bru/PycharmProjects/moabb/dataset/:/root/mne_data/"
+PIPELINE="/workdir/pipelines/"
```

Comment:
We should copy pipelines directory from git or a mirror site.

Reply:
This is an interesting question. Every time we build Docker, we copy a version of moabb into the image that theoretically has the most up-to-date pipeline.

However, every time we run Docker, we don't necessarily pull the latest pipeline version, which would involve cloning the repository on every run.

Some possible solutions:

1. Integrate the build and run scripts to ensure we are always synchronized with the latest version; 

Note: we may have to change the build script to be self-contained (no hard preferences).

2. Add a small bash script to download the moabb URL before running the run.

3. Change the run.py script to accept a link instead of a path and force that to make it download a folder.

What do you prefer @sylvchev?

Reply:
number 2.